lib/MusicBrainz/Server/Controller/Role/Alias.pm

```diff
@@ -49,6 +52,14 @@ sub aliases : Chained('load') PathPart('aliases')
         entity => $entity->TO_JSON,
     );
 
+    if ($ENTITIES_WITH_RELATIONSHIP_CREDITS{$self->{entity_name}}) {
+        my @relationship_credits = $c->model('Relationship')->load_credits(
+            $entity->entity_type,
+            [ $entity->id ],
```

Comment:
This looks wrong because the function accepts `@ids`, not `$ids`. _But_, it should work accidentally (I didn't test it yet) because you're only passing one array ref, and because of the way `load_credits` is written: since you don't try to take a reference to `@ids`, only use it as a list, the single array ref in the list matches up with each `any(?)`.

Since we don't seem to have any need for `load_credits` to accept more than one ID at present, I'd suggest changing the signature to only take one `$id`, and remove the `any`s (which in some cases I've seen can measurably speed up queries, even if you're only passing one value to the `any`).